services/market/shared/market/sources/common.js

```diff
@@ -13,8 +13,12 @@
  * Removal or modification of this copyright notice is prohibited.
  *
  */
-const { getMarketPrices } = require('./market');
+const moment = require('moment');
+
+const validateEntries = (entries, allowRefreshAfter) => entries.some(
```

Comment:
Ensure that `entries` is an instance of `Array` before invoking `some` method.

```suggestion
const validateEntries = (entries, allowRefreshAfter) => Array.isArray(entries) && entries.some(
```